RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
@jan-kaspar - geometry scenario for current year is http://cmslxr.fnal.gov/source/Geometry/CMSCommonData/python/cmsExtendedGeometry2016aXML_cfi.py?v=CMSSW_8_1_X_2016-06-06-2300

There are a few catches, however. This is taken from DB, so a GT update would be needed. Usually, RECO step would not have access to a DDCompactView.